frontend/src/lib/components/HelpButton/HelpButton.tsx

```diff
@@ -135,18 +122,33 @@ export function HelpButton({
                         >
                             Create an issue on GitHub
                         </LemonButton>
+                        {billing?.current_total_amount_usd > 0 && (
```

Comment:
⚠️ There's a TypeScript error here.
BTW why check spend and not the plan? Spend may be zero at the beginning of a billing period, right?

Reply:
Fair point. I've changed it to be the total predicted spend, and set a minimum of 1.6k/month spend.